text/0207-context-scoped-attributes.md

```diff
@@ -0,0 +1,245 @@
+
+Context-scoped attributes should be thought of equivalent to adding the attribute directly to each single telemetry item it applies to.
+
+### Comparing Context-scoped attributes to Baggage
+
+Context-scoped attributes and [baggage](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md) have two major commonalities:
+
+* Both are "bags of attributes"
+* Both are propagated identically in-process (Baggage also lives on the Context).
+
+However, the use cases are very different: Baggage is meant to transport app-level data along the path of execution, while execution-scoped attributes are annotations for all telemetry in the program execution "below" this point.
+
+This also explains the functional & API differences:
+
+* Most important point: Baggage is meant to be propagated; in fact, this is the main use case of baggage. While a propagator for Context-scoped attributes would be implementable, it would break the intended meaning by extending the scope of attributes to called services and would also raise many security concerns.
+* Baggage is both readable and writable for the application, while execution-scoped attributes, like all other telemetry attributes, are write-only.
+* Baggage only supports string values.
+* Baggage is not available to telemetry exporters (although e.g., a SpanProcessor could be used to change that), while that's the whole purpose of execution-scoped attributes.
+
+### Comparing Context-scoped attributes to Scope Attributes
+
+Context-scoped attributes and the recently added [Scope attributes](https://github.com/open-telemetry/oteps/pull/201) have these commonalities:
+
+* Both have "scope" in the name
+* Both apply a set of telemetry attributes to a set of telemetry items (those in their "scope")
+
+What is different is the scope. While Scope attributes apply to all items emitted by the same Tracer, Meter or LogEmitter (i.e., typically the same telemetry-implementing unit of code),
+the scope of Context-scoped attributes is determined at runtime by the Context.
+
+In practice, these scopes will [often be completely orthogonal](#scope-and-context), i.e., as the Context is "propagated" in-process through a single
+service, there will often be exactly one span per Tracer (e.g., one span from the HTTP server instrumentation as the request arrives,
+and another one from a HTTP client instrumentation, as a downstream service is called)
+
+I suggest calling Scope Attributes emitter scope attributes for better distinguishability.
```

Comment:
Can you clarify this sentence? I think you mean to rename `Scope Attributes` to `Emitter Scope Attributes`?

Reply:
Scope is shorthand for [Instrumentation Scope](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-scope). We can call them Instrumentation Scope Attributes.